### ISSUE TITLE
vnc-recorder is not able to write recordings Linux permissions?

### ISSUE THREAD

**Issue by kwbr** [testcontainers/testcontainers-java#518]
I always get errors, that the recording can not be moved to the target destination. The reason is, that the file to be moved does not exists. The screen.flv is not created. I use Linux as my build server, under MacOS it works.

```
Could not move recording file from .tmp-volume-1513089992384/screen.flv to target/FAILED-SeleniumContainerTest-simplePlainSeleniumTest-20171212-154641.flv
```

Wild guess: I read about permission problems writing to the host filesystem under Linux.

**Comment by kiview** [comment]
How is your build server setup? If the file is still there, can you please make a `ls -la` on the file and on your `/target` dir?

Sounds like something that might happen if the Docker container writes the files with root user id and so other users aren't allowed to move/delete it.

**Comment by kwbr** [comment]
The movie file is never created. The vcr-container dies instantly. The tests are run with an unprivileged user who has write permissons to the target/ directory
```
1a25bef353cf        richnorth/vnc-recorder:latest            "flvrec.py -o /recor…"   3 seconds ago       Exited (139) 1 second ago
```

It runs the Docker entrypoint with the following arguments

```
[-o /recording/screen.flv -P /recording/password vnchost 5900]
```

Quick googling for exit code 139: segfault?

Any idea how I can debug this further?

**Comment by kwbr** [comment]
BTW I can connect to the selenium/standalone-chrome-debug with VNC and see the test running.

**Comment by bsideup** [comment]
Hi @kwbr,
Good news - it's already fixed in #526 :) We will merge &release it soon, you can also try it yourself with jitpack:
https://www.testcontainers.org/usage.html#maven-dependencies

**Comment by kwbr** [comment]
Thanks, I'll test it out as soon as it's merged and released. Just subscribed for notifications on the issue.

**Comment by rnorth** [comment]
I suspect that, if not already fixed, #888 may finally fix this.
